posthog/api/decide.py

```diff
@@ -159,5 +159,19 @@ def get_decide(request: HttpRequest):
                 on_permitted_recording_domain(team, request) or not team.recording_domains
             ):
                 response["sessionRecording"] = {"endpoint": "/s/"}
+
+            source_files = PluginSourceFile.objects.filter(
+                plugin__pluginconfig__team=team,
+                plugin__pluginconfig__enabled=True,
+                filename="web.ts",
+                status=PluginSourceFile.Status.TRANSPILED,
+            ).all()
+            if len(source_files) > 0:
+                response["inject"] = []
```

Comment:
Nit: This could be a list comprehension:

```
response["inject"] = [
  {"id": source_file.plugin_id, "source": source_file.transpiled, "payload": {}} 
  for source_file in source_files
]